src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,9 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        if (number == 1 || number == 0) {
```

Comment:
why have you used two conditions here?

Reply:
because one of tests requires factorial from zero, which is 1, just like factorial from 1. using two conditions here makes code shorter
should I put  "(if (number == 0)" to another if block?

Reply:
you got it right :)